client/lib/ajax.js

```diff
@@ -1,45 +1,151 @@
-import $ from 'jquery';
+const $ = require('jquery');
```

Comment:
Small note: jquery is a huge library to import just for its ajax functionality. This looks great and functional, but FYI if you were looking for places to optimize it would make sense to use a library that is specifically used to make ajax calls like axios or fetch.

Reply:
This was part of the client side, legacy code. I plan to update this soon.

Reply:
Would you like me to update it now?